src/App.tsx

```diff
@@ -1,10 +1,50 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './Components/TodoList';
+import { TodoFilter } from './Components/TodoFilter';
+import { SortType } from './types/SortType';
+import { Error } from './Components/Error';
 
-const USER_ID = 0;
+const USER_ID = 10390;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+  const [sort, setSort] = useState(SortType.All);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const completedTodo = todos.filter(todo => todo.completed === true);
+  const activeTodo = todos.length - completedTodo.length;
+
+  const getFiltered = (filter: SortType) => {
```

Comment:
Wrap it with `useMemo` for optimization
```suggestion
  const filteredTodos = useMemo(() => {
```